samples/wifi/sta/Kconfig

```diff
@@ -8,6 +8,14 @@ source "Kconfig.zephyr"
 
 menu "Nordic Sta sample"
 
+config MAXIMUM_ITERATIONS
+	int "Number of iterations for logging the station connection status"
+	default 100
+
+config STATUS_POLLING_MS
```

Comment:
Even this is not something user understands or cares, this was mainly for our debugging.